share/locale/languages.xml

```diff
@@ -37,6 +37,7 @@
   <language code="it" name="Italiano" />
   <language code="ja" name="日本語" />
   <language code="ka" name="ქართული" />
+  <language code="kab" name="ⵜⴰⵇⴱⴰⵢⵍⵉⵜ" />

```

Comment:
Another issue: this string doesn't display in MuseScore's languages selection dialog, not on Windows 7 and 10 at least. It also doesn't show, when opening that file with IE11 (on Windows 7, works on Windows 10, there also with Edge), but does in QtCreator and in GitHub for Windows.
Might be some font issue on Windows, with MuseScore's default menu font?